Sources/TimeFreeze.swift

```diff
@@ -40,9 +54,11 @@ struct TimeFreeze {
         if rint(currentBoot) - rint(previousBoot) != 0 {
             self.uptime = currentUptime
             self.timestamp = currentTimestamp
+            self.isAdjustedForReboot = true
```

Comment:
I'd store the NTP sync time on a state as well as into disk using the same mechanism we use for storing times, right after sync'ing (and update it every time we hit an NTP server). I'd get rid of this flag; this way on a reboot we won't restore that date (reminding nil for the period of time while we are sync'ing the clock)

Reply:
doing so would make the age less trustful, which we want to avoid. discussion below.